compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -979,8 +979,12 @@ fn link_natively<'a>(
                      but `link.exe` was not found",
                 );
                 sess.note_without_error(
-                    "please ensure that VS 2013, VS 2015, VS 2017, VS 2019 or VS 2022 \
-                     was installed with the Visual C++ option",
+                    "please ensure that Visual Studio 2013 or later, or Build Tools \
```

Comment:
This should be changed to 2017+ at least?

https://github.com/rust-lang/rust/blob/2d46584fae1acc74566bf49fce976fe509a38f5f/README.md?plain=1#L152-L156